### ISSUE TITLE
Magento 2 CE, RC2, Release Notes/Known Issues

### ISSUE THREAD

**Issue by tanberry** [magento/magento2#2347]
<h2 id="known-devbeta">Magento 2.0.0, Release Candidate 2 (CE)</h2>

<h3 id="known-devbeta">Known Issues</h3>
This page discusses known issues in Magento 2.0.0, Release Candidate 2 (CE).

We have identified the following known issues in this release:

 
 * <a href="#customtime">Custom Timezone setting does not work properly for all stores</a>
 * <a href="#catcountdownticker">Catalog Event countdown ticker should not be cached</a>
 * <a href="#brokenbundle">Bundle products in sample data is broken</a>
  * <a href="#missingmethod">Eway Method is missing from payment functionality matrix report</a>
  * <a href="#datacollector">Relative path in data collector backup does not work in nginx</a>
  * <a href="#vimeobutton">Vimeo video player does not contain fullscreen mode button and video title</a>
 * <a href="#productfield">"Does this have a weight?" Yes/No Product field not working for bundle product</a>
 * <a href="#customadminurl">Custom Admin URL with https is enabled with errors = "No"</a>
 * <a href="#datacollector">Data collector backup not working on some configurations</a>
 * <a href="#specialchars">Inline Translations not working correctly for phrases with special character</a>
 * <a href="#brokenrefresh">Reports Refresh Lifetime Statistics feature not working</a>
 * <a href="#sessionlock">Session locking</a>
 * <a href="#M2reindex">Reindexing from Magento Admin not available</a>
 * <a href="#nocartrule">Cart price rule is not applied</a>
 * <a href="#pinchzoom">Pinch zoom not working on Storefront Gallery</a>
  * <a href="#SQLerror">SQL error after search in Billing Agreements grid in Admin</a>
 * <a href="#chgpagelayout">If you change page layout on grid, then the node of this page will be removed from hierarchy and Custom Theme will be changed to "Magento Blank"</a>
 * <a href="#catpermit">More specific CatalogPermission doesn't override general CatalogPermission</a>
 * <a href="#nodiscount">Totals are wrong if discount is applied when placing order from Admin</a>
 * <a href="#cachingnoncacheable">Uncacheable blocks being cached</a>
 * <a href="#customstorecode">Product API does not work properly for non-default store code</a>
 * <a href="#xdebug">xdebug</a>

<h4 id="known-devbeta-customtime">Custom Timezone setting does not work properly for all stores</h4>

When installing Magento on more than one website, or with multiple stores, all time zones revert to the first time zone you set during installation.  

<h4 id="known-devbeta-catcountdownticker">Catalog Event countdown ticker should not be cached</h4>

Magento currently caches the Catalog Event countdown ticker. If you enable the ticker on the Product Category and Product View pages,  the Countdown counter on those pages will display the wrong time.

<h4 id="known-devbeta-brokenbundle">Bundle products in sample data is broken</h4>

Bundle products in the Magento Sample data are not available from the Magento frontend. It is available from the Magento Admin with no title. 

<h4 id="known-devbeta-missingmethod">Eway Method is missing from payment functionality matrix report</h4>
The Eway method is missing from the Payment Functionality Matrix section of the Configuration report. If you generate a new configuration report from System > System Report, the Eway method is missing.

<h4 id="known-devbeta-datacollector">Relative path in data collector backup does not work in nginx</h4>

The Data Collector backup returns relative, not absolute, paths.

<h4 id="known-devbeta-vimeobutton">Vimeo video player does not contain fullscreen mode button and video title</h4>

The video title and fullscreen mode button are missing from the Vimeo video player. 

<h4 id="known-devbeta-productfield">"Does this have a weight?" Yes/No Product field not working for bundle product</h4>

When creating a bundle product from the Products > Catalog page, Magento displays this prompt, "Does this have a weight?" Even if you answer yes, Magento does not provide a field for entering the product weight. 

When creating a bundle product from the Products > Catalog page, Magento displays this prompt, "Does this have a weight?" If you answer no, Magento provides a field for entering the product weight.

<h4 id="known-devbeta-customadminurl">Custom Admin URL with https is enabled with errors = "No"</h4>

The Admin panel shows multiple errors when you log in after running the "Custom Admin URL with https" test. Styles are lost, and the browser console displays multiple errors. 

<h4 id="known-devbeta-missingmethod">Data collector backup not working on some configurations</h4>

You cannot back up the data collector from the System > Support > Data Collector page when running the following configuration: 

* CentOS

* Apache 2.2.15

* MySQL Server Version 5.6.26
 
* PHP Version 5.5.28

<h4 id="known-devbeta-specialchars">Inline Translations not working correctly for phrases with special characters</h4>

Magento's inline translation mechanism does not correctly translate phrases containing escaped  characters,  such as &, <, or >. Translation tables incorrectly display entries that contain escaped characters. For example, the translation table displays "Find Partners & Extensions" as "Find Partners &amp;amp; Extensions".

<h4 id="known-devbeta-brokenrefresh">Reports Refresh Lifetime Statistics  feature not working</h4>

You cannot run Refresh Lifetime Statistics from the Open Reports page.

<h4 id="known-devbeta-sessionlock">Session locking</h4>

We recommend you use memcached for session storage. The Redis session handler in the `phpredis` PHP extension does not support session locking, which might cause issues with distributed systems and applications that rely on Ajax.

<h4 id="known-devbeta-M2reindex">Reindexing from Magento Admin not available</h4>
Unlike Magento 1, Magento 2 does not enable you to reindex using the Magento Admin. Instead, you must set up `cron` (the UNIX task scheduler). In Magento 2, `cron` is even more important; it is used for the following features:

* Catalog price rules
* Newsletters
* Generating Google sitemaps
* Customer Alerts/Notifications (product price change, product back in stock)
* Reindexing
* Private sales (Magento EE only)
* Automatic updating of currency rates
* All Magento e-mails (including order confirmation and transactional)

<h4 id="known-devbeta-nocartrule">Cart price rule is not applied </h4>

The Cart Price rule is not applied as a Payment method condition. As a result,  your storefront will not display any discount you create using the Cart Price rule. 

<b>Work-around</b>: Use an alternate form of setting discount conditions. 

<h4 id="known-devbeta-pinchzoom">Pinch zoom not working on Storefront Gallery</h4>

This issue affects only Android phones and tablets running Google Chrome. 

Pinch zoom is not working on Android phones and tablets running running Google Chrome. Magento does not display the pinch-zoomed image of the Storefront Gallery  and does not properly display the image on the Android tablet after you change display orientation from portrait to landscape.  

<h4 id="known-devbeta-SQLerror">SQL error after search in Billing Agreements field </h4>

An SQL error occurs, and your search fails, when you specify a date in the Updated At field of the Sales > Billing Agreements page.

<h4 id="known-devbeta-chgpagelayout">Changing page layout on grid</h4>

Changing the home page of a CMS page results in its using the Magento Blank theme instead of the correct theme. In addition, the expected hierarchy does not display.

<h4 id="known-devbeta-catpermit">More specific CatalogPermission doesn't override general CatalogPermission</h4>

Specific catalog permissions do not override the general catalog permissions. 

<h4 id="known-devbeta-nodiscount">Totals are wrong if discount is applied when placing order from Admin</h4>
Magento does not correctly calculate an order total if you place an order from the Admin panel. 

<h4 id="known-devbeta-cachingnoncacheable">Uncacheable blocks being cached</h4>

Magento is currently caching pages that contain layouts that have non-cacheable blocks.  

<h4 id="known-devbeta-customstorecode">Product API does not work properly for non-default store code</h4>
If you submit a REST <code>POST  /V1/products</code> request using a store code that is not <code>default</code>, the result is displayed on the default store.

<h4 id="known-devbeta-xdebug">xdebug</h4>
If you use the optional PHP extension `xdebug`, you can encounter exceptions:

*   During installation 
*   Accessing either the Magento Admin or storefront after a successful installation 

Sample exception:

    Fatal error: Maximum function nesting level of '100' reached, aborting!

<b>Work-around</b>: To resolve this issue, you can:

*   Disable the `xdebug` extension.
*   Set the value of `xdebug.max_nesting_level` to a value of 200 or more. For more information, see <a href="http://xdebug.org/docs/basic#max_nesting_level" target="_blank">xdebug documentation</a>.

After you change the configuration of or disable `xdebug`, restart Apache:

*   CentOS: `sudo service httpd restart`
*   Ubuntu: `sudo service apache2 restart`

**Comment by IceReaper** [comment]
Any news on the "Cart price rule is not applied" problem? Still occuring in 2.0.2 here...

**Comment by jfrontain** [comment]
We are investigating the issue and will respond as soon as we can. Thanks for following up on this issue.

**Comment by tanberry** [comment]
long ago published, sorry we never closed it! thanks @veloraven for cleaning this up!

**Comment by vincentwinkel** [comment]
Sorry but still exists on Magento 2.0.14.
I have to do this trick as workaround: https://magento.stackexchange.com/questions/86674/block-with-cachable-false-not-rendered-on-product-view-page/90130#90130
